xlconnect.Rmd

```diff
@@ -0,0 +1,544 @@
+Using the package [`XLConnect`](https://github.com/miraisolutions/xlconnect) will allow you to read, write and manipulate MS Excel files from within R.
+
+XLConnect's main features:
+
+- read & write Excel worksheets & named regions
+
+- create, remove, rename, clone & hide / unhide worksheets
+
+- add graphs
+
+- specify / apply cell styles, column width & row height
+
+- define behavior when error cells are encountered
+
+- force formula recalculation when workbooks are opened
+
+- others: merge / unmerge cells, set auto-filters
+
+- etc.
+
+## Installation
+
+XLConnect is cross-platform and runs in all operating systems that support Java. Besides R and a JRE (Java Runtime Environment), nothing else is needed, even an installation of MS Excel is NOT required.
+
+```{r , echo = TRUE, eval = FALSE}
+# Install XLConnect
+install.packages("XLConnect")
+
+# Load the XLConnect package and get ready to use it
+require(XLConnect)
+```
+
+## Basic functions: write & read an Excel sheet
+
+### Writing Excel files
+
+```{r , echo = TRUE, eval = FALSE}
+# Load Excel workbook - either .xls or .xlsx. Create if not existing
+wb <- loadWorkbook("xlconnect-demo/XLConnectDemo.xlsx", create = TRUE)
+
+# Create a Sheet within an Excel workbook
+createSheet(wb, name = "CHF")
+
+# Write into a sheet
+# Write Swiss Francs data frame into CHF sheet
+writeWorksheet(wb, swissfranc, sheet = "CHF", startRow = 2, startCol = 2)
+
+# Save workbook - this actually writes the file
+saveWorkbook(wb)
+```
+
+The steps above can be executed in a single call using the `writeWorksheetToFile()` function.
+```{r , echo = TRUE, eval = FALSE}
+# Write and save a data frame into a specific sheet of a workbook
+writeWorksheetToFile("xlconnect-demo/XLConnectDemo.xlsx", 
+                     data = swissfranc, sheet = "OneCall", 
+                     startRow = 2, startCol = 2)
+```
+
+The data frame can also be written in a named region using the single call `writeNamedRegionToFile()` function. This will create the sheet (if necessary) and named region name based on the formula argument and call writeNamedRegion instead of writeWorksheet.
+```{r , echo = TRUE, eval = FALSE}
+# Write and save a data frame into a specific named region of a workbook
+writeNamedRegionToFile("xlconnect-demo/XLConnectDemo.xlsx", 
+                       data = swissfranc, name = "SwissFrancName", 
+                       formula = "NamedRegionEx!$B$2")
+```
+
+### Reading from  Excel files
+
+```{r , echo = TRUE, eval = FALSE}
+# Load Excel workbook - either .xls or .xlsx. Create if not existing.
+wb <- loadWorkbook("xlconnect-demo/XLConnectDemo.xlsx", create = TRUE)
+
```

Comment:
```suggestion
```
This is misleading, as it relies on the fact that this workbook has already been created and filled with data. create = TRUE does not overwrite and if it did, the lines below would fail.
Either remove entirely or remove create = TRUE and align comment above